src/party_menu.c

```diff
@@ -1200,8 +1207,6 @@ static void HandleChooseMonSelection(u8 taskId, s8 *slotPtr)
                 TryEnterMonForMinigame(taskId, (u8)*slotPtr);
             break;
         default:
-        case PARTY_ACTION_ABILITY_PREVENTS:
-        case PARTY_ACTION_SWITCHING:
             PlaySE(SE_SELECT);
```

Comment:
Leave these if it's possible for `gPartyMenu.action` to be either value here, to help people searching for where they might be read.